### ISSUE TITLE
Accessibility (WCAG level A) The version table on the specific packages' pages doesn't use scope attributes on th elements

### ISSUE THREAD

**Issue by claycompton** [NuGet/NuGetGallery#1185]
1. Go to https://nuget.org/packages/jQuery/ and view source.
2. Scrutinize the HTML for the version table and observe the markup for the <th> elements in the page.

Result:  None of them have scope attributes.
Expected:  All of them should have scope="col" attributes on them to assist screen readers.

**Comment by johnataylor** [comment]
We have a new gallery design.
